pos_product_mergeable_line/static/src/js/models.js

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright (C) 2023-Today: GRAP (http://www.grap.coop)
+@author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define("pos_product_mergeable_line.models", function (require) {
+    "use strict";
+
+    const {Orderline} = require("point_of_sale.models");
+    const Registries = require("point_of_sale.Registries");
+
+    const PosProductMergeableLineOrderline = (OriginalOrderline) =>
+        class extends OriginalOrderline {
+            can_be_merged_with(orderline) {
+                console.log("==================");
```

Comment:
you can remove console log :)

Reply:
indeed ! Thanks ! 
Note : I thought that console.logs were forbidden and that the CI would be red, but that's not the case.